yb-voyager/src/cp/yugabyted/yugabyted_test.go

```diff
@@ -0,0 +1,169 @@
+package yugabyted
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4/pgxpool"
+	_ "github.com/lib/pq" // PostgreSQL driver
+	"github.com/stretchr/testify/assert"
+	controlPlane "github.com/yugabyte/yb-voyager/yb-voyager/src/cp"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/testutils"
+	"github.com/yugabyte/yb-voyager/yb-voyager/testcontainers"
+)
+
+func TestDatabaseTablesYugabyteD(t *testing.T) {
```

Comment:
Name it someyhing which tests that we are comparing the table definition 
`TestYugabyteDTableSchemaValidation` ?